pkg/api/http/server.go

```diff
@@ -8,24 +8,55 @@ import (
 	"github.com/cloudwego/hertz/pkg/common/utils"
 	"github.com/cloudwego/hertz/pkg/protocol/consts"
 	"time"
+	"glide/pkg/api"
+	"sync"
+	"log/slog"
+	
 )
 
 type HTTPServer struct {
 	server *server.Hertz
 }
 
+type ServerManager struct {
+	httpServer *HTTPServer
+	shutdownWG *sync.WaitGroup
+}
+
 func NewHttpServer(config *HTTPServerConfig) (*HTTPServer, error) {
 	return &HTTPServer{
 		server: config.ToServer(),
 	}, nil
 }
 
+// This func has the routes for the server
 func (srv *HTTPServer) Run() error {
-	srv.server.GET("/health", func(ctx context.Context, c *app.RequestContext) {
-		c.JSON(consts.StatusOK, utils.H{"healthy": true})
-	})
+    srv.server.GET("/health", func(ctx context.Context, c *app.RequestContext) {
+        c.JSON(consts.StatusOK, utils.H{"healthy": true})
+    })
+
+    srv.server.POST("/chat", func(ctx context.Context, c *app.RequestContext) {
```

Comment:
Are you still going to work on a Unified API GEP? That should give a bigger picture on what Glide API design could look like. In turn, that will produce a unified API schema that we could validate all request against like in [Hertz examples](https://github.com/cloudwego/hertz-examples/tree/main/binding). And then that unified validated schema is going to be passed to the corresponding client.